tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
```

Comment:
For the name of the file to be consistent with the one above it can be `literature_suggestion.feature` and `test_literature_suggestion.py`